catalog/core/catalog-core-camelcomponent/src/test/java/ddf/camel/component/catalog/CatalogComponentFrameworkTest.java

```diff
@@ -688,9 +686,7 @@ public void configure() throws Exception {
         }
 
         UpdateRequest updateRequest = new UpdateRequestImpl(metacardIds, metacards);
-        UpdateResponse updateResponse = new UpdateResponseImpl(updateRequest,
-                new HashMap(),
-                updates);
+        new UpdateResponseImpl(updateRequest, new HashMap(), updates);
```

Comment:
can remove the this line, the line above and the `when` line below, they don't affect the assertions